modules/components/pages/inputs/redpanda.adoc

```diff
@@ -0,0 +1,619 @@
+= redpanda
+// tag::single-source[]
+:type: input
+:page-beta: true
+:categories: ["Services"]
+
+component_type_dropdown::[]
+
+Consumes topic data from one or more Kafka brokers.
+
+ifndef::env-cloud[]
+Introduced in version 4.39.0.
+endif::[]
+
+Use this connector in conjunction with the xref:components:outputs/redpanda.adoc[`redpanda` output], if you need to move topic data between Redpanda clusters, or other Apache Kafka clusters.
```

Comment:
Is the accurate?

Reply:
No, the redpanda output isn't necessary, they can both be used independently and with any other plugins.